mod.ts

```diff
@@ -61,12 +67,27 @@ const defaultOptions: LoadOptions = {
 export class Coffee {
   private loadOptions: LoadOptions = defaultOptions;
   private isLoaded = false;
+  private fileExts = [
+    "json",
+    "yml",
+  ];
 
   runtimeAPI: RuntimeAPI = new DenoAPI();
   parsers: { [k: string]: Parser } = {
     json: (t: string) => JSON.parse(t),
+    yml: (t: string) => this.ymlReader(t),
   };
   configs: Configs = {};
+  private ymlReader(t: string): Configs {
```

Comment:
It's Okay, but it could be better to move this method outside of Coffee class, in a separate file in `src` folder

Reply:
i got, bro